.pre-commit-config.yaml

```diff
@@ -49,6 +49,8 @@ repos:
     -   id: trailing-whitespace
         name: trailing-whitespace-markdown
         types: [markdown]
+    -   id: check-merge-conflict
+        args: [--assume-in-merge]
```

Comment:
Normally this only runs if it's in a `git merge` situation. That meant it missed the error until I added this flag.